pristine-zen-garden.html

```diff
@@ -1,157 +1,145 @@
-<ul>
-  <li>
-  <a href="/205/" class="design-name">spring360</a> by <a href="http://www.medialab360.com" class="designer-name">Rene Hornig</a>
-  </li>         <li>
-  <a href="/204/" class="design-name">Withering Beauty</a> by     <a href="http://www.wduffy.co.uk/" class="designer-name">William Duffy</a>
-  </li>         <li>
-  <a href="/203/" class="design-name">Tiny Blue</a> by            <a href="http://www.timovirtanen.com/" class="designer-name">Timo Virtanen</a>
-  </li>         <li>
-  <a href="/202/" class="design-name">Retro Theater</a> by            <a href="#" class="designer-name">Eric Rogé</a>
-  </li>         <li>
-  <a href="/201/" class="design-name">Lily Pond</a> by            <a href="#" class="designer-name">Rose Thorogood</a>
-  </li>         <li>
-  <a href="/200/" class="design-name">Icicle Outback</a> by           <a href="http://www.timovirtanen.com/" class="designer-name">Timo Virtanen</a>
-  </li>         <li>
-  <a href="/199/" class="design-name">CSS ZEN ARMY</a> by           <a href="http://www.niceguy.com" class="designer-name">Carl Desmond</a>
-  </li>         <li>
-  <a href="/198/" class="design-name">The Original</a> by           <a href="#" class="designer-name">Joachim Shotter</a>
-  </li>         </ul>
-</nav>
-</div>
-
-<div class="design-archives" id="design-archives">
-<h3 class="archives">Archives:</h3>
-<nav role="navigation">
-<ul>
-<li class="next">
-<a href="/214/page3">
-Next Designs <span class="indicator">&rsaquo;</span>
-</a>
-</li>
-<li class="previous">
-<a href="/214/page1/">
-<span class="indicator">&lsaquo;</span> Previous Designs              </a>
-</li>
-<li class="viewall">
-<a href="http://www.mezzoblue.com/zengarden/alldesigns/" title="View every submission to the Zen Garden.">
-View All Designs              </a>
-</li>
-</ul>
-</nav>
-</div>
-
-<div class="zen-resources" id="zen-resources">
-<h3 class="resources">Resources:</h3>
-<ul>
-<li class="view-css">
-<a href="/214/214.css" title="View the source CSS file of the currently-viewed design.">
-View This Design&#8217;s <abbr title="Cascading Style Sheets">CSS</abbr>            </a>
-</li>
-<li class="css-resources">
-  <a href="http://www.mezzoblue.com/zengarden/resources/" title="Links to great sites with information on using CSS.">
-<abbr title="Cascading Style Sheets">CSS</abbr> Resources           </a>
-</li>
-<li class="zen-faq">
-  <a href="http://www.mezzoblue.com/zengarden/faq/" title="A list of Frequently Asked Questions about the Zen Garden.">
-<abbr title="Frequently Asked Questions">FAQ</abbr>           </a>
-</li>
-<li class="zen-submit">
-  <a href="http://www.mezzoblue.com/zengarden/submit/" title="Send in your own CSS file.">
-Submit a Design           </a>
-</li>
-<li class="zen-translations">
-<a href="http://www.mezzoblue.com/zengarden/translations/" title="View translated versions of this page.">
-Translations            </a>
-</li>
-  </ul>
-  </div>
-</div>
-</aside>
-
-
-</div>
-
-</body>
+   <head>
```

Comment:
head html and body can be on the same line so your whole file can be indented left by one